test/steps/v2_steps.py

```diff
@@ -2685,6 +2775,24 @@ def check_atomic_transaction_composer_response(context, returns):
             assert result.decode_error is None
 
 
+@then('The app should have returned ABI types "{abiTypes:MaybeString}".')
+def check_atomic_transaction_composer_return_type(context, abiTypes):
+    expected_tokens = abiTypes.split(":")
+    for i, expected in enumerate(expected_tokens):
+        result = context.atomic_transaction_composer_return.abi_results[i]
+        assert result.decode_error is None
+
+        if not result or not expected_tokens[i]:
```

Comment:
Why would `not result` ever return true?

Reply:
👍